lib/flutter_calendar_carousel.dart

```diff
@@ -665,9 +663,9 @@ class _CalendarState extends State<CalendarCarousel> {
           page = page - 1;
         }
 
-        this.setState(() {
-          _startWeekday = dates[page].weekday;
-          _endWeekday = dates[page + 1].weekday;
+        setState(() {
```

Comment:
You know how to format dart. Thanks.

Reply:
Hum... Sorry but I'm kind of new to dart so I don't get your comment. Where is the problem concerning the format? I've looked on the api doc and some flutter videos from the flutter team and it seems that the `this` is mandatory when calling `setState` (at least they don't use it in their videos or on the website).
Same for class variables, it seems you don't have to refer to the object instantiated with `this` to read and write them.
If it's not about that and you've set some rules concerning the format to follow in order to PR, could you share me them?